postgres-persistence/src/main/java/com/netflix/conductor/dao/postgres/PostgresQueueDAO.java

```diff
@@ -190,54 +191,85 @@ public void processAllUnacks() {
         logger.trace("processAllUnacks started");
 
         getWithRetriedTransactions(tx -> {
-            String LOCK_TASKS = "SELECT message_id FROM queue_message WHERE popped = true AND (deliver_on + (60 ||' seconds')::interval)  <  current_timestamp FOR UPDATE SKIP LOCKED";
+            String LOCK_TASKS = "SELECT queue_name, message_id FROM queue_message WHERE popped = true AND (deliver_on + (60 ||' seconds')::interval)  <  current_timestamp limit 1000 FOR UPDATE SKIP LOCKED";
```

Comment:
Do you think there's value in making the batch size configurable? Did you test with different values? 1000 seems like a reasonable default.

Reply:
@mactaggart I had thought about that but I can't think of a situation where someone would actually figure out that for their database there is a better value to use and there are already so  many properties  that I didn't want to add another one.